megabeast/plotting/plot_graphic_model.py

```diff
@@ -0,0 +1,93 @@
+from graphviz import Digraph
+import numpy as np
+
+__all__ = ["create_graphic_model", "plot_graphic_model"]
+
+
+def create_graphic_model(nodes, edges, type):
+    """
+    Create a graphic model given nodes and edges
+
+    Parameters
+    ----------
+    nodes : dict
+        for each node {key, text, math}
+    edges : dict
+        for each edge {key, text, math}
+    type : str [default="text"]
+        "text" for a verbose version, "math" for a compact version
+    """
+    mod = Digraph()
+    if type == "math":
+        tindx = 1
+    else:
+        tindx = 0
+    for ckey in nodes.keys():
+        if ckey == "Like":
+            cstyle = "filled"
+        else:
+            cstyle = None
+        mod.node(ckey, nodes[ckey][tindx], style=cstyle)
+
+    for ckey in edges.keys():
+        for cval in np.atleast_1d(edges[ckey]):
+            mod.edge(ckey, cval)
+
+    return mod
+
+
+def plot_graphic_model(type="text", savefig="png"):
```

Comment:
Codacy found an issue: [Redefining built-in 'type'](https://app.codacy.com/manual/kgordon/megabeast/pullRequest?prid=6302069)